### PR TITLE
Register Subdomain: custom

### PR DESCRIPTION
**Pull request by d14her**
Added the domain: custom

### CODE REVIEW

File: domains/cu3t0m.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Cu3t0m",
+        "email": "kickstart330@gmail.com"
+      },
+      "record": {
+        "CNAME": "https://d4e5069a-53af-43e9-9ef6-0c4818cf70da.id.repl.co"
```
Comment (WilliamDavidHarrison) [inline domains/cu3t0m.json:8]:
```suggestion
        "CNAME": "d4e5069a-53af-43e9-9ef6-0c4818cf70da.id.repl.co"
```

File: domains/custom.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "Cu3t0m",
+        "email": "kickstart330@gmail.com"
+      },
+      "record": {
+        "CNAME": "https://d4e5069a-53af-43e9-9ef6-0c4818cf70da.id.repl.co"
```
Comment (WilliamDavidHarrison) [inline domains/custom.json:8]:
```suggestion
        "CNAME": "d4e5069a-53af-43e9-9ef6-0c4818cf70da.id.repl.co"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please add content to your website

**Comment by WilliamDavidHarrison** [comment]
Only a hello world website and no response.

If you want a domain please finish your website and make a new PR.

**Pull request closed without merging**